js/components/interface/query/queryBuilder.js

```diff
@@ -1288,14 +1025,21 @@ define(function (require) {
             return { __html: resultsItem.verboseLabel };
           };
 
-          return (value === index
-                        && <Typography component="div" key={index}>
-                          <div className="result-verbose-label" dangerouslySetInnerHTML={getVerboseLabelMarkup()}></div>
-                          <div className="clearer"></div>
-                          <Griddle columns={this.state.resultsColumns} results={resultsItem.records}
-                            showFilter={true} showSettings={false} enableInfiniteScroll={this.state.infiniteScroll} resultsPerPage={this.state.resultsPerPage} bodyHeight={(window.innerHeight - 280)}
-                            useGriddleStyles={false} columnMetadata={this.state.resultsColumnMeta} />
-                        </Typography>
+          return (focusTabIndex === index
+                            && <Typography component="div" key={index}>
```

Comment:
@ddelpiano something upsets me in this indentation 😬

Reply:
already discussed, blame the linter or @rodriguez-facundo in case 😂